### PR TITLE
Docs/address warnings

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -91,7 +91,17 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = [
+        'datatypes/**',
+        'developer_guide/**',
+        'get_started/**',
+        'howto/installation_more/index.rst',
```
Comment (sphuber) [inline docs/source/conf.py:98]: I think Giovanni added this one, do you remember what the idea here was? Where should this be added at some point?

File: docs/source/conf.py [hunk 2]
```diff
@@ -91,7 +91,17 @@
...
+        'import_export/**',
+        'querying/**',
```
Comment (sphuber) [inline docs/source/conf.py:100]: Isn't this already migrated? The queryhelp and intro files are empty at least and can be deleted I reckon. Also the content of `append.rst` is either commented out or is content that is already addressed in `howto/data.rst` or `topics/database.rst`. So I think we can just remove the whole thing. The other file `backend.rst` I would also just delete. Minimal content and not actually we should recommend to users.

File: docs/source/howto/installation.rst [hunk 3]
```diff
@@ -236,7 +236,7 @@ However, optimal performance at that scale might require some tweaks to the AiiD
 
 Here are a few general tips that might improve the AiiDA performance:
 
-  1. :ref:`Move the Postgresql database<how-to:installation:more:move_postgresql>` to a fast disk (SSD), ideally on a large partition.
+  1. Move the Postgresql database to a fast disk (SSD), ideally on a large partition.
```
Comment (sphuber) [inline docs/source/howto/installation.rst:239]: I guess this was his intention for that file he just forgot to include it in the tree. Is there a place we can put this advanced content? Should we just inline it below the itemization?

**Review by sphuber — APPROVED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (csadorf) [reply docs/source/conf.py:98]: We didn't know where to put it.

File: docs/source/conf.py (hunk 2 above)
Comment (csadorf) [reply docs/source/conf.py:100]: I might have missed empty files. In that case, please feel free to just delete them.

File: docs/source/howto/installation.rst (hunk 3 above)
Comment (csadorf) [reply docs/source/howto/installation.rst:239]: I guess we can inline it for now.

**Pull request merged**